cmd/shfmt/Dockerfile

```diff
@@ -1,9 +1,13 @@
-FROM golang:1.13.5-alpine3.10
+FROM golang:1.13.5-alpine3.11 as build
```

Comment:
i'd uppercase the 'AS' as it is a keyword (you did below in the alpine case)

Reply:
Yup, my bad. I'll fix it.